### DISCUSSION TITLE
Tooltip only showing on certain elements

### DISCUSSION THREAD

**Discussion by agostino-c** [GrapesJS/grapesjs#5830] [Q&A] [answered]
Hello all,
Please delete if this is in the incorrect category, I don't want to open it as a bug because I'm sure it's due to user-error. I am building an HTML builder with GrapesJS, and I got the basic scaffolding of my editor done. Everything's looking and feeling great so far, however, it seems when I hover over for tooltips, only my blocks display a tooltip. I've done everything from trying to mimic similar classes/ids to the blocks but to no avail. I even tried adding a "title" attribute to the panel button config.  I'm just thinking at this point it may be better to implement my own tooltips for buttons/objects. Has anyone else had any experience with this? I will attach images of the difference as well as my JS for my panel and block setup for reference. TIA!

![2](https://github.com/GrapesJS/grapesjs/assets/165092191/43957c8d-c4c9-40b4-8753-3d95fb89de10)
![1](https://github.com/GrapesJS/grapesjs/assets/165092191/fb982c73-2c15-4e68-b932-d9ad5d6753d5)

```
editor.Panels.addPanel({
    id: 'basic-actions',
    el: '.panel_basic_actions',
    buttons: [
        {
            id: 'visibility',
            active: true,
            className: 'btn-toggle-borders',
            label: '<i class="fa-solid fa-border-all"></i>',
            command: 'sw-visibility',
        },
        {
            id: 'clear-content',
            className: 'tooltip-component',
            title: "Clear",
            label: '<i class="fa-solid fa-eraser"></i>',
            command: 'core:canvas-clear',
        },
        {
            id: 'undo-button',
            className: 'btn-toggle-borders',
            label: '<i class="fa-solid fa-rotate-left"></i>',
            command: 'core:undo',
        },
        {
            id: 'redo-button',
            className: 'btn-toggle-borders',
            label: '<i class="fa-solid fa-rotate-right"></i>',
            command: 'core:redo',
        },
        {
            id: 'feedback-button',
            className: 'btn-toggle-borders',
            label: '<i class="fa-solid fa-clipboard-question"></i>',
            command: 'feedback-press',
        },
    ]
});
```

**Comment by artf** [comment, marked as answer]
There is always an option to implement your own UI

**Reply by agostino-c** [reply]
Perfect, thank you. I just wanted to make sure it wasn't just user-error. Appreciate the quick response.
